### PR TITLE
Release 0.20.2

### PR DESCRIPTION
**Pull request by Pat-Lafon**
As noted in #864 and #848, it's probably safe to cut a new release? There have been a few changes since 0.20.1 but most of them are minor or documentation related. I've noted the new features in the updated release notes as I understand them.

### CODE REVIEW

**Review by youknowone — APPROVED** [review]
I am sorry for not tracking these issues recently. I will publish it once @dburgener approve it. (as agreement of 0.20.2 instead of 0.21.0)

@nikomatsakis Could you grant cargo publish permission to @Pat-Lafon? That will make publish schedule better managed.

File: Cargo.toml [hunk 1]
```diff
@@ -14,7 +14,7 @@ resolver = "2"
 repository = "https://github.com/lalrpop/lalrpop"
 authors = ["Niko Matsakis <niko@alum.mit.edu>"]
 license = "Apache-2.0 OR MIT"
-version = "0.20.1" # LALRPOP
+version = "0.20.2"                                # LALRPOP
```
Comment (yannham) [inline Cargo.toml:17]:
```suggestion
version = "0.20.2" # LALRPOP
```

Comment (yannham) [reply Cargo.toml:17]: Or is there a reason for this spacing? It's not very important but I personally think any kind of visual alignement (beyond basic indentation) in code will eventually not not work out as expected...

File: doc/calculator/Cargo.toml [hunk 2]
```diff
@@ -1,12 +1,15 @@
 [package]
 name = "calculator"
-version = "0.20.1"
+version = "0.20.2"
 authors = ["Niko Matsakis <niko@alum.mit.edu>"]
-workspace = "../.." # <-- We added this and everything after!
+workspace = "../.."                             # <-- We added this and everything after!
```
Comment (yannham) [inline doc/calculator/Cargo.toml:5]:
```suggestion
workspace = "../.." # <-- We added this and everything after!
```

Feel free to ignore those if you have a good reason to add all those spaces.

File: doc/nobol/Cargo.toml [hunk 3]
```diff
@@ -3,10 +3,13 @@ name = "nobol"
 version = "0.1.0"
 edition = "2021"
 authors = ["Felix S Klock II <pnkfelix@pnkfx.org>"]
-workspace = "../.." # <-- We added this and everything after!
+workspace = "../.."                                 # <-- We added this and everything after!
```
Comment (yannham) [inline doc/nobol/Cargo.toml:6]:
```suggestion
workspace = "../.." # <-- We added this and everything after!
```

**Review by yannham — APPROVED** [review]

**Comment by yannham** [comment]
I don't have publish rights either, unfortunately.

File: Cargo.toml (hunk 1 above)
Comment (Pat-Lafon) [reply Cargo.toml:17]: No, I think it was just autoformatting... I'm not sure why it did that

Comment (yannham) [reply Cargo.toml:17]: It's a strange formatting rule :sweat_smile:

Comment (Pat-Lafon) [reply Cargo.toml]: Sorry for the noise, it was my EvenBetterToml extension on VsCode that wants to align comments by default.

**Review by dburgener — APPROVED** [review]

**Comment by dburgener** [comment]
Sorry about the delay in responding here.  I'm am out of the office for an extended leave for a bit, so my responses are slow.

I think this is good.  The incompatibility noted in 0.20.1 has been resolved, and I had retested that specific repo at the time.  I had hoped to get better automated test coverage to avoid those sorts of issues in the future, and I still hope to do that, but I don't think we need to gate 0.20.2 on that.

**Comment by yannham** [comment]
3 approves, I'm going to proceed and merge then. @youknowone , I guess you're the only one to have to power to proceed with publishing from here :slightly_smiling_face:

**Comment by youknowone** [comment]
Thank you all! I published 0.20.2

**Comment by Pat-Lafon** [comment]
> @nikomatsakis Could you grant cargo publish permission to @Pat-Lafon? That will make publish schedule better managed.

I wonder if there is a more sustainable solution in the form of automation than handing out permissions. Given both the recent supply chain attacks but also the ever improving tooling like cargo-semvar-checks and https://github.com/MarcoIeni/release-plz.

**Pull request merged**